<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1241</width>
      <height>781</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1241</width>
      <height>781</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border: 0px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="StepPopup" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>661</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="TeachPopup" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>661</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="symptomsList" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>271</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="instructionsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>641</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1309</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
